gears/src/rest/pagination.rs

```diff
@@ -25,6 +25,23 @@ pub fn parse_pagination(pagination: Pagination) -> (u32, u8) {
     (page, limit)
 }
 
+impl From<core_types::query::request::PageRequest> for Pagination {
+    fn from(
+        core_types::query::request::PageRequest {
+            key: _,
+            offset,
+            limit,
+            count_total: _,
+            reverse: _,
+        }: core_types::query::request::PageRequest,
+    ) -> Self {
+        Self {
+            offset: Some(offset as u32),
```

Comment:
These casts can wrap around, which is probably not the desired behaviour.

Reply:
I'll try to change it there

Reply:
Okay. I tried to make it `u64`, but we depend on tendermint in [this](https://github.com/NYBACHOK/gears/blob/c97a2a209acf0b192859781d2c6deaa767544451/gears/src/rest/handlers.rs#L69) code, so I have chosen to set max values if couldn't narrow number